test/unittests/execute_floating_point_test.cpp

```diff
@@ -51,25 +51,25 @@ TYPED_TEST(execute_floating_point_types, test_values_selftest)
     EXPECT_EQ(std::size(TV::positive_all()), TV::num_positive + TV::num_nans);
 
     for (const auto nan : TV::positive_nans())
-        EXPECT_TRUE(std::isnan(nan));
+        EXPECT_TRUE(nan.is_nan());
 
     EXPECT_TRUE(FP<TypeParam>{*TV::canonical_nan}.is_canonical_nan());
 
     for (const auto nan : TV::positive_noncanonical_nans())
     {
-        EXPECT_TRUE(std::isnan(nan));
+        EXPECT_TRUE(nan.is_nan());
         EXPECT_FALSE(FP<TypeParam>{nan}.is_canonical_nan());
     }
 
     for (const auto p : TV::positive_all())
-        EXPECT_EQ(std::signbit(p), 0);
+        EXPECT_EQ(p.sign_bit(), 0);
 
     const auto& ordered = TV::ordered();
     auto current = ordered[0];
     EXPECT_EQ(current, -FP<TypeParam>::Limits::infinity());
     for (size_t i = 1; i < ordered.size(); ++i)
     {
-        EXPECT_LT(current, ordered[i]);
+        EXPECT_LT(current.as_float(), ordered[i].as_float());
```

Comment:
And this of the one adding `as_float`?

Reply:
This is changed because test values are `FP` objects now.